cmake/toolchains/libfuzzer.cmake

```diff
@@ -1,2 +1,2 @@
 # Require libfuzzer specific flags
-set(CMAKE_CXX_FLAGS "-O1 -gline-tables-only -fsanitize=address -fsanitize-address-use-after-scope -fsanitize=fuzzer-no-link -stdlib=libstdc++")
+set(CMAKE_CXX_FLAGS "-O1 -gline-tables-only -fsanitize=address -fsanitize-address-use-after-scope -fsanitize=fuzzer-no-link -stdlib=libstdc++ -fopenmp=libgomp")
```

Comment:
This will be needed for the fuzzing builds to choose the correct openmp implementation once the docker image is updated and the build chooses the static Z3 library (which uses gcc openmp).